flutter_vlc_player/CHANGELOG.md

```diff
@@ -1,11 +1,3 @@
-## 4.0.3
-Update VLCKit for iOS and iOS. Cleanup example Pod file. Clean up example gradle. Credits to Mitch Ross (https://github.com/mitchross).
-Removed dispose calls on VlcPlayerController from VlcPlayer.
-Fix argument-less functions throwing FlutterMethodNotImplemented.
-
-## 4.0.2
-* Update Cocoapods version for VLCkit on iOS. This fixes issues with iOS 12 and Simulators. Credits to Mitch Ross (https://github.com/mitchross).
-
```

Comment:
Could you please restore those versions for the reference?

Reply:
yea was lost in the git switch over. Ill add em back no problem.

Reply:
fixed